### DISCUSSION TITLE
Third party plugins install, but building reports that they are not installed

### DISCUSSION THREAD

**Discussion by stevekirby** [mkdocs/mkdocs#3472] [Q&A] [answered]
I can install plugins without any obvious problems, but as soon as I enable one in the .yml file, the site breaks and terminal says:

`ERROR   -  Config value 'plugins': The "mkdocs-video" plugin is not installed`

 I'm on Mac OS 13.5.2, MKdocs v1.5.3 and Python v3.10.

If I run `pip3 freeze` I do see the plugins listed:

```Babel==2.13.1
certifi==2023.7.22
charset-normalizer==3.3.2
click==8.1.7
colorama==0.4.6
ghp-import==2.1.0
idna==3.4
Jinja2==3.1.2
lxml==4.9.3
Markdown==3.5.1
MarkupSafe==2.1.3
mergedeep==1.3.4
mkdocs==1.5.3
mkdocs-autolinks-plugin==0.7.1
mkdocs-autorefs==0.5.0
mkdocs-bootswatch==1.1
mkdocs-gitbook==0.0.1
mkdocs-material==9.4.8
mkdocs-material-extensions==1.3
mkdocs-simple-blog==0.0.9
mkdocs-terminal==4.4.0
mkdocs-video==1.5.0
packaging==23.2
paginate==0.5.6
pathspec==0.11.2
platformdirs==4.0.0
Pygments==2.16.1
pymdown-extensions==10.4
python-dateutil==2.8.2
PyYAML==6.0.1
pyyaml_env_tag==0.1
regex==2023.10.3
requests==2.31.0
six==1.16.0
urllib3==2.1.0
watchdog==3.0.0
```

I've tried removing all Markdown extensions and using the default theme, and it still breaks. If I explicitly enable the `search` plugin, there's no problem.

Any idea what could be going wrong?

**Comment by Andre601** [comment]
Could you actually share your mkdocs.yml (The version that causes these errors)?

**Reply by stevekirby** [reply]
_file removed_
Thanks for taking a look

**Reply by Andre601** [reply]
I meant posting it in a code block here, not sending it as zip.

**Comment by pawamoy** [comment]
Make sure that the `mkdocs` command you're calling is the one from your virtualenv. You can do that by running `hash -r` once, which will clear the executable path cache. You can also explicitly call `python -m mkdocs`, which reduces the risk of using `mkdocs` from another environment.

**Reply by stevekirby** [reply]
Thanks. I've tried all those commands in zsh terminal without any errors, but my problem still persists.

**Comment by pawamoy** [comment, marked as answer]
I suggest to start from scratch again and progressively add the extensions and plugins you are using.

```bash
mkdir mydocs
cd mydocs
python -m venv .venv
. .venv/bin/activate
python -m pip install mkdocs mkdocs-video
python -m mkdocs new .
echo "plugins: [mkdocs-video]" >> mkdocs.yml
python -m mkdocs serve
```

This works without any issue.

From there, add your Markdown pages, the rest of the configuration in mkdocs.yml, etc.

**Reply by stevekirby** [reply]
I tried all that and at the last step I got this:
```
(.venv) steve@Steves-MacBook-Air mydocs % python -m mkdocs serve
INFO    -  Building documentation...
INFO    -  Cleaning site directory
INFO    -  Documentation built in 0.06 seconds

OSError: [Errno 48] Address already in use
```

**Reply by stevekirby** [reply]
Ok I started over completely from scratch with a different directory, and now it's working. Thanks.
